### ISSUE TITLE
pause when volume reaches 0

### ISSUE THREAD

**Issue by Turhvjbufv** [OxygenCobalt/Auxio#637]
### Description

Add an option to enable
Pause the music when the volume is lowered to zero and resume when it's heightened 

### Other implementations

Retro music, there a setting in there that does exactly what I described
https://github.com/RetroMusicPlayer/RetroMusicPlayer

### Benefit

Sometimes you don't want to pick up your phone and press pause, maybe because it's in your pocket or idk, and with that, just long pressing down volume button you can pause the music instead of the music continuing to play on zero volume

### Duplicates

- [X] I have checked this feature request for duplicates.
- [X] I have checked that this feature is not implemented in the [lastest version](https://github.com/OxygenCobalt/Auxio/releases).
- [X] I have checked the [Why Are These Features Missing?](https://github.com/OxygenCobalt/Auxio/wiki/Why-Are-These-Features-Missing%3F) page.
- [X] I agree to the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by midtsveen** [comment]
I have to disagree with the automatic pausing of the music when volume is lowered to 0% because the users already have the option to pause the music using the pause button 🤷

**Comment by breversa** [comment]
> I have to disagree with the automatic pausing of the music when volume is lowered to 0% because the users already have the option to pause the music using the pause button on the Phone/Headphones 🤷

The use case that @Turhvjbufv presents probably supposes that there’s no physical pause button on the headphones.

Also, even it there was, sometimes it’s unreachable (e.g. under clothing).

Also, what about playing music on the speakerphone, not with headphones?

In conclusion, I think there are enough uses cases where that feature request can add functionality. :-)

**Comment by Martysh12** [comment]
I'd definitely like to have this in Auxio. I often mute a song with the volume buttons when I need to talk with someone, but then I have to rewind the song. Plus, I feel like there's no real use case of letting a song continue playing while it's muted.

**Comment by OxygenCobalt** [comment]
Okay, this seems meaningfully high demand (and even I mute songs myself). Will add.

**Comment by dotdot65654** [comment]
Will this be on the next update @OxygenCobalt

**Comment by sebsebseb9b** [comment]
I hope this is on the next update @OxygenCobalt 

👍👍👍
